lib/python/rosie/ws_client_auth.py

```diff
@@ -379,7 +379,7 @@ def _prompt(self, is_retry=False):
             if self.username:
                 username = ""
 
-            prompt = self.PROMPT_USERNAME % {"prefix": self.prefix}
+            prompt = self.PROMPT_USERNAME % {"host": self.root}
```

Comment:
The variable is "self.root" here (rightly or wrongly), so we may as well use "root" as the key here.